src/components/PipelineRun/PipelineRunListView/pipelinerun-actions.tsx

```diff
@@ -79,15 +70,21 @@ export const usePipelinererunAction = (pipelineRun: PipelineRunKind) => {
           !componentError &&
           startNewBuild(component).then(() => {
             navigate(
-              `${PIPELINE_RUNS_LIST_PATH.createPath({ workspaceName: namespace, applicationName: component.spec.application })}?name=${component.metadata.name}`,
+              `${PIPELINE_RUNS_LIST_PATH.createPath({
+                workspaceName: namespace,
```

Comment:
Another topic(not related to the PR), do we need to migrate off the workspaceName in our codes, just change all workspaceName to namespaceName?

Reply:
yes, eventually we can